libs/utils/target_script.py

```diff
@@ -89,7 +89,11 @@ def push(self):
         :type actions: list(str)
         """
 
-        actions = ['set -e'] + self.commands + ['set +e']
+        self._pid_file = self._target.path.join(self._target.working_directory, 
```

Comment:
We already chatted about it offline, but I'll put it here as well: having to keep track of the process that is executing the script seems strange to me - in a perfect world, issuing a `kill()` via devlib should be enough.